src/components/ResourcesCards.js

```diff
@@ -137,6 +152,14 @@ function ResourcesCards() {
             description="Check out our SHPE UF: Info Slides to learn more about the programs, events, and opportunities SHPE UF has to offer!"
             extra={<a class="extra" href="https://28d1f378-e21f-4379-a063-11326d549c54.filesusr.com/ugd/4f1a8d_68ff9284c7dd40658aa324292ece1ac9.pdf" target="_blank">Click here to download the slides!</a>}
         />
+        {resources.map((resource) => (
+            <Card
+                image={<Image src={resourceImage} as="a" href={resource.link} target="_blank"/>}
+                header={<Grid><Grid.Column textAlign="center">{<a className="ui header" href={resource.link} target="_blank">{resource.title}</a>}</Grid.Column></Grid>}
+                description={resource.description}
+                extra={<a class="extra" href={resource.link} target="_blank">Access Here!</a>}
```

Comment:
I don't think we need the extra field, esp since the image and header link to the resource already